website/src/components/SearchPage/DownloadDialog/SequenceFilters.tsx

```diff
@@ -30,8 +30,9 @@ export interface SequenceFilter {
 
     /**
      * Return a map of keys to human-readable descriptions of the filters to apply.
+     * null values are maintained as null.
      */
-    toDisplayStrings(): Map<string, [string, string]>;
+    toDisplayStrings(): Map<string, [string, string | null]>;
```

Comment:
naive question - can the values ever be `undefined`? should we also add that case here?